src/ProductConstructionService/ProductConstructionService.Api/appsettings.json

```diff
@@ -11,5 +11,14 @@
         "QueuePollTimeout": "00:01:00",
         "MaxJobRetries": 3,
         "QueueMessageInvisibilityTime": "00:01:00"
+    },
+    "Kusto": {
+        "Database": "engineeringdata"
+    },
+    "GitHubAuthentication": {
+        "SaveTokens": true,
+        "CallbackPath": "/signin/github",
+        "UserAgentProduct": "",
```

Comment:
Possibly this should go instead of the `Maestro` value in the product name when initializing the GitHub client?

Reply:
I'm not sure what you mean by this

Reply:
We should have a value here and use it in
https://github.com/dotnet/arcade-services/pull/3338#discussion_r1504024738